testing/crosstool-ng/APKBUILD

```diff
@@ -25,6 +25,18 @@ build() {
 package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install
+
+	# bash completion has it's own package
+	rm -r "$pkgdir"/usr/share/bash-completion/
```

Comment:
It's perhaps a matter of preference, but isn't it easier to just move the files from `$pkgdir` to `$subpkgdir` rather than removing and 'reinstalling'?

Reply:
I have implemented the move as it was suggested. Thank you!